specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -1820,6 +1829,142 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionHostSubsetConfigurationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved subset configuration.",
+            "schema": {
+              "$ref": "#/definitions/SessionHostSubsetConfiguration"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SessionHostSubsetConfiguration"
+        ],
+        "operationId": "SessionHostSubsetConfigurations_Delete",
+        "description": "Remove a SessionHostSubsetConfiguration.",
+        "x-ms-examples": {
+          "SessionHostSubsetConfigurations_Delete": {
+            "$ref": "./examples/SessionHostSubsetConfigurations_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionHostSubsetConfigurationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully removed subset configuration.",
+            "schema": {
+              "$ref": "#/definitions/SessionHostSubsetConfiguration"
+            }
+          },
+          "204": {
+            "description": "Subset configuration does not exist."
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/sessionHostSubsetConfigurations": {
+      "get": {
```

Comment:
where is "put" (create)?

Reply:
was going to do that in the next iteration. This is to add the default subsetConfig which created through SHC create